frontend/src/components/Popup.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { Component, useState } from "react";
+import { withRouter, history } from "react-router";
+import Button from "../components/Button";
+import H1 from "../components/H1";
+import "./../index.css";
```

Comment:
the `index.css` is already imported in the index.js and thus will be bundled in the build by webpack, you don't have to import it here again

Reply:
deleted the import